benches/benches.rs

```diff
@@ -14,8 +14,6 @@
 
 //! An example of using simplectic noise
 
-#![feature(asm)]
-#![feature(core)]
 #![feature(test)]
```

Comment:
Cargo only tries to build this file when doing `cargo bench` and this file completely fails to work without that feature.